tests/unittest/cloud_governance/zombie_cluster/test_zombie_cluster_resources.py

```diff
@@ -103,31 +103,31 @@ def test_zombie_cluster_route_table():
     This method return all cluster route_table
     :return:
     """
-    print(zombie_cluster_resources.zombie_cluster_route_table())
+    assert len(zombie_cluster_resources.zombie_cluster_route_table()) >= 0
 
 
 def test_zombie_cluster_internet_gateway():
     """
     This method return all cluster internet_gateway
     :return:
     """
-    print(zombie_cluster_resources.zombie_cluster_internet_gateway())
+    assert  len(zombie_cluster_resources.zombie_cluster_internet_gateway()) >= 0
 
 
 def test_zombie_cluster_dhcp_option():
     """
     This method return all cluster dhcp_option
     :return:
     """
-    print(zombie_cluster_resources.zombie_cluster_dhcp_option())
+    assert len(zombie_cluster_resources.zombie_cluster_dhcp_option()) >= 0
 
 
 def test_zombie_cluster_vpc_endpoint():
     """
     This method return all cluster vpc_endpoint
     :return:
     """
-    assert len(zombie_cluster_resources.zombie_cluster_vpc_endpoint())
+    assert len(zombie_cluster_resources.zombie_cluster_vpc_endpoint()) >= 0
```

Comment:
we deleted all the zombie vpc_endpoints so it returns 0. 
assert 0 is an error.